README.md

```diff
@@ -1,4 +1,6 @@
 # PyoMyo
+
```

Comment:
When using java generics there is no need to specify the type in the assignment if it can be inferred by the compiler.